platform/jvm/JVMPlatformAssembler.kt

```diff
@@ -0,0 +1,339 @@
+package com.vaticle.bazel.distribution.platform.jvm
```

Comment:
`JVMPlatformAssembler` is the main component of the `assemble_jvm_platform` rule.